### DISCUSSION TITLE
how to modify the size of the internal thread pool when not using "async"?

### DISCUSSION THREAD

**Discussion by jinluyang** [fastapi/fastapi#10241] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
no need
```

### Description

if not using "async", fastAPI will use an internal threadpool. I guess it was the starlette threadpool. I want to change the size of the threadpool. I tried using "starlette.concurrency.threadpool_size=1", but it seems to be not working. How can I change the threadpool size??

### Operating System

Linux

### FastAPI Version

0.83.0

### Pydantic Version

1.10.2

### Python Version

3.8

**Comment by Kludex** [comment, marked as answer]
https://github.com/encode/starlette/issues/1724#issuecomment-1179063924
